src/test/java/io/polivakha/mojo/properties/WriteProjectPropertiesMojoTest.java

```diff
@@ -0,0 +1,137 @@
+package io.polivakha.mojo.properties;
+
+import org.apache.maven.model.Profile;
+import org.apache.maven.project.MavenProject;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Properties;
+
+import static org.junit.Assert.assertEquals;
+
+public class WriteProjectPropertiesMojoTest {
```

Comment:
Please. separate `WriteProjectProperties` tests from `WriteActiveProfileProperties` tests. Each class has to have one corresponding only to it test class

Reply:
No problem, thanks